src/utils/carousel.ts

```diff
@@ -33,6 +33,8 @@ export interface CarouselItem {
    * string to identify the title of the event contained in the widget
    */
   title: string
+
+  isoDate: string
```

Comment:
non conviene che sia già di tipo `Date`? giusto perché così rende più chiaro cos'è, dato che viene usato solo per venire istanziato in un oggetto Date anyway

Reply:
OK giusto ha più senso